Library/Services/Implementation/NfieldQuotaService.cs

```diff
@@ -45,7 +45,7 @@ public Task UpdateQuotaTargetsAsync(string surveyId, string eTag, IEnumerable<Qu
         {
             ValidateSurveyId(surveyId);
 
-            return Client.PutAsJsonAsync(EditingQuotaFrameTargetsUri(surveyId, eTag), targets)
+            return Client.PutAsJsonAsync(QuotaFrameTargetsUri(surveyId, eTag), targets)
                          .FlattenExceptions();
```

Comment:
It now points to the correct api controller. 
But there is another problem though. The type of etag is a string in SDK but in the controller it's a long

Reply:
urgh so messy

Reply:
@kefishev I talked with @joukevandermaas and it should be ok, it will be handled on the api side if etag is invalid.